translator/trace/protospan_to_jaegerthrift.go

```diff
@@ -0,0 +1,321 @@
+// Copyright 2018, OpenCensus Authors
```

Comment:
Is this meant to be part of the Jaeger exporter?

Reply:
May want to consider going directly to Jaeger grpc/protobuf instead of thrift. The grpc endpoint was merged recently, will be available in the next release as preview.